config/katello.migrations/160719114223-enableftp.rb

```diff
@@ -0,0 +1,7 @@
+# enable tftp service for foreman-proxy fixes #13546
+
+# foreman_proxy defaults
+answers['foreman_proxy'] = {
+  'tftp' => true,
+}
```

Comment:
This probably doesn't do what you intended - it will erase all of the users' other answers for foreman_proxy.

Best to do something like:

```ruby
answers['foreman_proxy']['tftp'] = true if answers['foreman_proxy'].is_a?(Hash)
```